Storage/Storage/Model/MIGRATIONS.md

```diff
@@ -2,6 +2,11 @@
 
 This file documents changes in the WCiOS Storage data model. Please explain any changes to the data model as well as any custom migrations.
 
+## Model 35 (Release 5.4.0.0)
```

Comment:
Probably it's better to merge this PR in release `5.5.0.0`?

Reply:
Hah yea, let's start following the no-merge-big-changes-on-Thursday/Friday-before-code-freeze guideline! I'll merge it after code freeze on my Tuesday next week

Reply:
updated in https://github.com/woocommerce/woocommerce-ios/pull/3078/commits/26050612ded63832d6dd6d1824a3869a8d34e5c9